client/my-sites/purchases/main.tsx

```diff
@@ -200,3 +209,67 @@ export function PurchaseCancelDomain( {
 		</Main>
 	);
 }
+
+export function BillingHistory() {
+	const selectedSiteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const translate = useTranslate();
+
+	return (
+		<Main className="purchases billing-history">
+			<MySitesSidebarNavigation />
+			<DocumentHead title={ translate( 'Billing History' ) } />
+			<PageViewTracker path="/purchases/billing-history" title="Billing History" />
+			<QueryBillingTransactions />
+			<FormattedHeader
+				brandFont
+				className="purchases__page-heading"
+				headerText={ translate( 'Billing' ) }
+				align="left"
+			/>
+			<BillingHistoryList siteId={ selectedSiteId } />
+			<CompactCard href="/me/purchases/billing">
+				{ translate( 'View all billing history and receipts' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'View billing history and receipts' )` *ES Score: 10*